lib/resources/user-collection.js

```diff
@@ -155,7 +155,8 @@ UserCollection.prototype.handle = function (ctx) {
       if(ctx.query.id || ctx.body.id) {
         this.save(ctx, done);
       } else {
-        this.store.first({username: ctx.body.username}, function (err, u) {
+        var usernameRegex = new RegExp(["^", ctx.body.username, "$"].join(""), "i");
```

Comment:
so this way u need to force username at account creation / logging in to be lowercased so u can just keep comparing text at account creation ?